content/01.main_content.md

```diff
@@ -1,35 +1,38 @@
   Numerous tools exist to algorithmically infer gender, nationality, and ethnicity using the feature most likely to be present in datasets: an individual's name [@doi:10.1093/pan/mpw001; @http://genderize.io/].
 
-Prediction models are not a panacea; several factors limit both their accuracy and utility.
+Prediction models are not a panacea, however; several factors limit both their accuracy at recapitulating self-reported information and their ability to address the underlying motivating questions of diversity audits.
   For instance, gender associations of a given name can vary by culture, potentially biasing gender predictions where additional information is not available [@https://aclanthology.org/U14-1021].
-  Also, most gender prediction models are trained on binary gender labels, which systematically overlooks and conceptually erases the representation and contributions of transgender, non-binary, and intersex individuals [@doi:10.1101/2020.10.12.336230].
+  Also, most gender prediction models are trained on binary gender labels, the use of which systematically overlooks and conceptually erases the representation and contributions of transgender, non-binary, and intersex individuals [@doi:10.1101/2020.10.12.336230].
 
 Proxy predictions of ethnicity via name origin are more difficult still; choosing categories to probabilistically predict on is non-trivial and difficult to discretize.
   Furthermore, there is no one-to-one mapping between having a name from a linguistic group and belonging to a minoritized or underrepresented group.
-  Colonialism, immigration, and structural racism have affected linguistic histories and inclusion or exclusion from scientific communities in complex ways that are nearly impossible to parse from names alone.
+  Colonialism, immigration, and structural racism have affected both linguistic histories and inclusion or exclusion from scientific communities in complex ways that are nearly impossible to parse from names alone.
   For instance, classifiers are usually unable to distinguish if names of Hispanic origin come from the Iberian Peninsula or from Latin America [@doi:10.1016/j.cels.2021.07.007].
+  If a target and background population had identical probabilistic proportions of Hispanic names, but the target group primarily consisted of Iberian individuals, underrepresentation of Latin American scientists would go unnoticed in an exclusively computational analysis.
 
-Recognizing the aforementioned shortcomings, we propose recommendations for the creation and deployment of automated auditing tools:
+Recognizing the aforementioned shortcomings, we propose the following recommendations for the creation and deployment of automated auditing tools:
   1. *Transparency.* Publicly provide all tools, code, and data used in the analysis.
   This enables public scrutiny to those being audited and those whose data are used.
```

Comment:
What do we mean by this? The public can scrutinize both who's being included and who's doing the auditing? Or that anyone whose data is used should be able to see the results? Or something else?

Reply:
I was intending 

1. the results and methods to get there to be available to the public so they can see if there is a bias problem in a specific organization. This is an argument against private internal audits that are never publicly shared
2. The data of the people who are included in the analysis (their name+predictions+other data) should be made available to them.

I could give a shot at re-wording it later if you like

Reply:
Yeah, I'd appreciate that. Still a little confused by the sentence.